host-bin/startxiwi

```diff
@@ -0,0 +1,93 @@
+#!/bin/sh -e
+# Copyright (c) 2015 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+set -e
+
+APPLICATION="${0##*/}"
+OPTS_ENTER=''
+OPTS_XIWI=''
+USAGE="$APPLICATION [options] CHROOT_APP [options]
+
+Wraps enter-chroot to launch a xiwi-app in a new windowed or tabbed session.
+You can launch any individual chroot graphical application in crouton windows.
+
+By default, it will log into the primary user on the first chroot found.
```

Comment:
`By default, it will use the primary user on the first xiwi-enabled chroot found.`